### PR TITLE
[Telemetry Table] Telemetry mode bug fixes

### LINKED ISSUE #7268: Improve telemetry table performance
**Is your feature request related to a problem? Please describe.**
Telemetry Tables use far too much CPU and memory, even with a single telemetry point at 10Hz.

eg. Create a 10 Hz SWG and drop it into a telemetry table. Open Chrome task manager and observe that CPU usage is 100+% and memory usage is over 500MB.

**Describe the solution you'd like**
I think that a 70% reduction in CPU utilization and at least 50% reduction in memory usage is achievable.

1. Tables should be limited to a fixed number of rows (Let's make it configurable, but 50 is probably a reasonable default). Rows beyond this can be accessed via paging. A fixed height should remove costly reflows, and limiting tables to 50 rows (ie. 50 telemetry values) should radically reduce memory usage.
2. Throttle table updates to 1Hz. when new data arrive, _or when time bounds change_ batch the changes and only update the table once per second.
3. De-reactify table rows. Right now they are Vue 3 proxy objects and this is costly. Right now rows are being made reactive by [being passed as a property to TelemetryTableRow](https://github.com/nasa/openmct/blob/2dc13887371aaace93ec889407d63d320652cd4d/src/plugins/telemetryTable/components/TableComponent.vue#L228). There's no need for this, [`rowIndex`](https://github.com/nasa/openmct/blob/2dc13887371aaace93ec889407d63d320652cd4d/src/plugins/telemetryTable/components/TableComponent.vue#L224) should be all that we need to trigger reactivity.

- [ ] Throttle table updates, de-reactify rows (2, 3) PR: #7392 
- [ ] Table performance mode (1) PR: #7399

### PR DESCRIPTION
**Pull request by jvigliotta**
Closes #7268

### Describe your changes:

Specifically addresses these issues: https://github.com/nasa/openmct/issues/7268#issuecomment-1998560136
- will set any existing tables to the defaults for TelemetryMode (default mode is limited)
- fixes error on save
- fixes telemetry mode and persist mode changes not working when changed at the same time

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [ ] Is this a [notable change](../docs/src/process/release.md) that will require a special callout in the release notes? For example, will this break compatibility with existing APIs or projects that consume these plugins?

### Author Checklist

* [x] Changes address original issue?
* [ ] Tests included and/or updated with changes?
* [x] Has this been smoke tested?
* [x] Have you associated this PR with a `type:` label? Note: this is not necessarily the same as the original issue.
* [x] Have you associated a milestone with this PR? Note: leave blank if unsure.
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?

### CODE REVIEW

File: src/plugins/telemetryTable/components/TableComponent.vue [hunk 1]
```diff
@@ -595,23 +596,35 @@ export default {
     },
     handleConfigurationChanges(changes) {
       const { rowLimit, telemetryMode, persistModeChange } = changes;
```
Comment (davetsay) [inline src/plugins/telemetryTable/components/TableComponent.vue:598]: If rowLimit shrinks, do we have to re-request?

**Review by davetsay — CHANGES_REQUESTED** [review]

File: src/plugins/telemetryTable/components/TableComponent.vue (hunk 1 above)
Comment (jvigliotta) [reply src/plugins/telemetryTable/components/TableComponent.vue:598]: SOLID question...

Comment (jvigliotta) [reply src/plugins/telemetryTable/components/TableComponent.vue:598]: For now yes.

**Review by davetsay — APPROVED** [review]
Flawless

**Pull request merged**